doc/authentication-backends/PKI-authentication-module.md

```diff
@@ -0,0 +1,24 @@
+## Overview
+
+PKI authentication backend is compatible only with SASL mechanisms, that are based on client certificates.
+Currently there is only one such mechanism: `SASL EXTERNAL`.
+
+It extracts Common Name from a certificate and returns it as a username part in JID.
```

Comment:
It extracts the Common Name from a certificate and returns it as a username part in JID.